docs/loading-profiles.md

```diff
@@ -0,0 +1,37 @@
+# Loading Profiles
+
+If you store all of your profiles in the same folder with a simple [barrel file](https://github.com/basarat/typescript-book/blob/master/docs/tips/barrel.md).
+
+```text
+└── src
+   └── maps
+      ├── foo.ts
+      ├── bar.ts
+      ├── index.ts
+      └── primitives
+        ├── string.ts
+        ├── number.ts
+        ├── boolean.ts
+        └── index.ts
+```
+
+Kitbag Mapper provides an easy way to to automatically loading profiles in this directory.
+
+```ts
+import mapper, { loadProfiles } from '@kitbag/mapper'
+import * as maybeProfiles from '@/maps'
```

Comment:
I feel like `maybeProfiles` is a little weird but I also don't have a good suggestion for a better name

Reply:
yeah that reads better to me. But I don't feel strongly either way.